### PR TITLE
SEO-175680-JS-UG-page-with-redirects-links

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes on the redirect links. Please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

**Comment by Edberg-Vasan** [comment]
rebuild

File: js/Chart/Exporting.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The exporting of the chart is done using the [`exportSettings`](../api/ejchart#m
 
 In client-side rendered chart can be exported as PNG image or as SVG file.The type of export is controlled by using the [`type`](../api/ejchart#members:exportsettings-type)property of the exportSettings.
 
-* Export as PNG - The chart can be exported to image when it is rendered in HTML5 Canvas.To render a chart in canvas, set the [`enableCanvasRendering`](../api/ejchart#members:enablecanvasrendering) option to *true*. To export the chart, you can use the [`export`](../api/ejchart#methods:export) method of the chart. Refer to the online [`KB for exporting`](https://www.syncfusion.com/kb/5045) to know more about chart exporting. 
+* Export as PNG - The chart can be exported to image when it is rendered in HTML5 Canvas.To render a chart in canvas, set the [`enableCanvasRendering`](../api/ejchart#members:enablecanvasrendering) option to *true*. To export the chart, you can use the [`export`](../api/ejchart#methods:export) method of the chart. Refer to the online [`KB for exporting`](https://support.syncfusion.com/kb/article/4957/how-to-export-the-chart-in-javascript) to know more about chart exporting.
```
Comment (Yvone-Atieno) [inline js/Chart/Exporting.md:19]: Check api links, they are redirecting

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: js/Chart/Exporting.md (hunk 1 above)
Comment (Edithabich) [reply js/Chart/Exporting.md:19]: resolved

File: js/Chart/Exporting.md [hunk 2]
```diff
@@ -10,13 +10,13 @@ api : /api/js/ejchart
 
 # Exporting in JavaScript Chart
 
-The exporting of the chart is done using the [`exportSettings`](../api/ejchart#members:exportsettings) property. The chart  export can be done in both client-side and in server-side.This can be modified by setting values to the property [`mode`](../api/ejchart#members:exportsettings-mode) in exporting. Default value for mode is client. 
+The exporting of the chart is done using the [`exportSettings`](https://help.syncfusion.com/api/js/ejchart#members:exportsettings) property. The chart  export can be done in both client-side and in server-side.This can be modified by setting values to the property [`mode`](https://help.syncfusion.com/api/js/ejchart#members:exportsettings-mode) in exporting. Default value for mode is client.
```
Comment (Yvone-Atieno) [inline js/Chart/Exporting.md:13]: why have added https.
Use this ../api/js/ejchart#members:exportsettings-mode

File: js/Chart/Exporting.md [hunk 3]
```diff
@@ -10,13 +10,13 @@ api : /api/js/ejchart
...
+The exporting of the chart is done using the [`exportSettings`](https://help.syncfusion.com/api/js/ejchart#members:exportsettings) property. The chart  export can be done in both client-side and in server-side.This can be modified by setting values to the property [`mode`](https://help.syncfusion.com/api/js/ejchart#members:exportsettings-mode) in exporting. Default value for mode is client. 
 
 ## Client-side Exporting
 
-In client-side rendered chart can be exported as PNG image or as SVG file.The type of export is controlled by using the [`type`](../api/ejchart#members:exportsettings-type)property of the exportSettings.
+In client-side rendered chart can be exported as PNG image or as SVG file.The type of export is controlled by using the [`type`](https://help.syncfusion.com/api/js/ejchart#members:exportsettings-type)property of the exportSettings.
```
Comment (Yvone-Atieno) [inline js/Chart/Exporting.md:17]: Check all api links and change

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: js/Chart/Exporting.md (hunk 2 above)
Comment (Edithabich) [reply js/Chart/Exporting.md:13]: Resolved

File: js/Chart/Exporting.md (hunk 3 above)
Comment (Edithabich) [reply js/Chart/Exporting.md]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**